.github/workflows/gh-pages.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: |
           cd results
           mkdir build-results
-          gsutil cp gs://${{ secrets.GCP_STORAGE_BUCKET }}/*.json.gz build-results
+          gsutil cp ://${{ secrets.GCP_STORAGE_BUCKET }}/*.json.gz build-results
```

Comment:
Why is this change required?

Reply:
Previously the CI run on push to Antmicro GH complained that this is illegal URL specification: https://github.com/antmicro/fpga-tool-perf/runs/5657033538?check_suite_focus=true

But with this change I have a different error message, see there: https://github.com/antmicro/fpga-tool-perf/runs/5657933114?check_suite_focus=true

So I'm actually puzzled now.

Reply:
Hmm we may try leaving the `gs` prefix there as it was previously working. I think that we can skip the whole CI check and merge it after the change, as this change would not affect the tests at all.

Reply:
Alright, I reverted to the `gs` prefix.